06-files-input-and-output/remove_files.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import os
+
+def find_files_in_folder(path, extention, file_max_size):
+    file_list = []
+
+    for current_folder, subfolders, file_names in os.walk(path):
+        for file_name in file_names:
+            full_path = os.path.abspath(os.path.join(current_folder, file_name))
+            file_size = os.path.getsize(full_path)
+
+            if file_name.lower().endswith(extention) and file_size < file_max_size:
+                file_list.append(full_path)
+
+    return file_list
+
+def main():
+    folder = "little pics"
+    full_path = os.path.join(os.path.dirname(__file__), folder)
```

Comment:
Extract a variable for the file's directory.
Add a print stating which path are you searching files in.